### PR TITLE
Reformat all tables

### PR DESCRIPTION
**Pull request by staltz**
I went through everything and clarified all tables, added examples, more links. Would appreciate review from more than 1 person.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,144 +1,185 @@
-# SSB binary field encodings
+# SSB Binary Field Encodings
 
 Shortname: SSB-BFE
 
 Definition of binary encodings for common types such as message ids
-and feed ids to be used in binary feed formats. This can be seen as an
-extension of the types defined in [TFK].
+and feed ids to be used in binary feed formats.
```
Comment (mycognosist) [inline README.md:6]: ids -> IDs

File: README.md [hunk 2]
```diff
@@ -1,144 +1,185 @@
...
 
 ```
   @6CAxOI3f+LUOVrbAl0IemqiS7ATpQvr9Mdw9LC4+Uv0=.ed25519
   │└─────────────────────┬────────────────────┘└───┬──┘
- sigil           base64 encoded key             suffix
+ sigil           base64 encoded data            suffix
 ```
 
-### Message type
+Its BFE encoding are the following bytes displayed in hexadecimal:
```
Comment (mycognosist) [inline README.md:52]: are -> is

"Its BFE encoding is the following bytes..."

File: README.md [hunk 3]
```diff
@@ -1,144 +1,185 @@
...
-|------|-------------|-------------|---------------|
-| 3    | 0           | curve25519  |               |
+```
+  02 00 4b bf 82 c0 73 3a 75 9f 6c 8b 95 67 e1 fb 69 93 f9 7f 95 d6 d1 32 ac f3 26 8b 3e b2 96 5b 59 fe
+   │  │ └────────────────────┬────────────────────────────────────────────────────────────────────────┘
+type  │                    data
+     format
+```
+
+### Diffie-hellman formats
```
Comment (mycognosist) [inline README.md:121]: Diffie-hellman -> Diffie-Hellman

**Review by mycognosist — APPROVED** [review]
LGTM. I first read for correctness of content and couldn't spot any errors. The second reading was for formatting / grammar and I suggested some small changes.

**Comment by staltz** [comment]
Thanks! I'll fix those welcome corrections

**Review by arj03 — APPROVED** [review]

**Pull request merged**